config/environments/development.rb

```diff
@@ -60,4 +60,9 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # In dev allow connections from any host
+  Rails.application.configure do
+    config.hosts.clear
```

Comment:
What's this for?

Reply:
See https://www.fngtps.com/2019/rails6-blocked-host/. I didn't think there was any way we could guess what everyone's host name in the development environment would be, so just allowed all (which would be equivalent to what prior versions of Rails).